stdpopsim/generic_models.py

```diff
@@ -0,0 +1,87 @@
+# generic_models
+import msprime
+import stdpopsim.models as models
+
+
+#########
+# Generic Demographic Models
+#
+#
+########
+
+class generic_constant_size(models.Model):
```

Comment:
These are classes, so we should follow the SnakeCase convention like the other classes. -> GenericConstantSize

Reply:
ugh. my bad here